<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite ignored="0" hostname="DESKTOP-FU6EMRP" failures="0" tests="4" name="iiitd.nrl.evalapp.YouTubeTests" time="0.000" errors="1" timestamp="24 Sep 2021 11:22:40 GMT" skipped="3">
  <testcase classname="iiitd.nrl.evalapp.YouTubeTests" name="copyVideoTime" time="0.000">
    <skipped/>
  </testcase> <!-- copyVideoTime -->
  <testcase classname="iiitd.nrl.evalapp.YouTubeTests" name="launchCap" time="0.000">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at iiitd.nrl.evalapp.MyDatabase.addTestResult(MyDatabase.java:140)
	at iiitd.nrl.evalapp.YouTubeTests.launchCap(YouTubeTests.java:100)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at iiitd.nrl.evalapp.MainLauncher.main(MainLauncher.java:209)
]]>
    </error>
  </testcase> <!-- launchCap -->
  <testcase classname="iiitd.nrl.evalapp.YouTubeTests" name="playTest" time="0.000">
    <skipped/>
  </testcase> <!-- playTest -->
  <testcase classname="iiitd.nrl.evalapp.YouTubeTests" name="channelTest" time="0.000">
    <skipped/>
  </testcase> <!-- channelTest -->
</testsuite> <!-- iiitd.nrl.evalapp.YouTubeTests -->
